src/templates/components/staffProfileSideBarNav/index.tsx

```diff
@@ -34,58 +33,21 @@ const StaffProfileSideBarNav: React.FC<SidebarNavProps> = ({
       className="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav"
     >
       <div>
-        <button
-          className="va-btn-close-icon va-sidebarnav-close"
-          type="button"
-          aria-label="Close this menu"
-        ></button>
```

Comment:
This is another vestigial thing. This button does nothing and doesn't even show at small screen sizes since the whole sidebar gets dropped

Reply:
🫠